File: src/main/java/com/pisphere/market/LiquidityPoolManager.java
package com.pisphere.market;

import java.util.HashMap;
import java.util.Map;

public class LiquidityPoolManager {
    private Map<String, Double> liquidityPools;

    public LiquidityPoolManager() {
        liquidityPools = new HashMap<>();
    }

    public void addLiquidity(String asset, double amount) {
        liquidityPools.put(asset, liquidityPools.getOrDefault(asset, 0.0) + amount);
        System.out.println("Added liquidity for " + asset + ": " + amount);
    }

    public double getLiquidity(String asset) {
        return liquidityPools.getOrDefault(asset, 0.0);
    }
}

File: src/main/python/market/liquidity_pool_manager.py
class LiquidityPoolManager:
    def __init__(self):
        self.liquidity_pools = {}

    def add_liquidity(self, asset, amount):
        self.liquidity_pools[asset] = self.liquidity_pools.get(asset, 0.0) + amount
        print(f"Added liquidity for {asset}: {amount}")

    def get_liquidity(self, asset):
        return self.liquidity_pools.get(asset, 0.0)
